ape-config.yaml

```diff
@@ -35,90 +33,6 @@ solidity:
     - "@fx-portal/contracts=fx-portal/v1.0.5"
     - "@threshold/contracts=threshold/v1.2.1"
 
-deployments:
-  polygon:
-    mainnet:
-      - contract_type: DAI
-        address: "0x8f3Cf7ad23Cd3CaDbD9735AFf958023239c6A063"
-      - contract_type: fx_child
-        address: "0x8397259c983751DAf40400790063935a11afa28a"
```

Comment:
It's important not to lose track of FxPortal official addresses. I guess we can add them in a later PR when we add the PolygonRoot and PolygonChild scripts, but mentioning just in case. Same for DAI contracts.

Reply:
I think the new methodology is great. I was just pointing out to some addresses that we should collect later